### PR TITLE
Bugfix - configurable phase banner survey

### PR DESCRIPTION
**Pull request by joefitter**
* Added phaseBannerSurvey prop as this should only show for external users

### CODE REVIEW

File: ui/views/base.jsx [hunk 1]
```diff
@@ -60,7 +61,8 @@ const Layout = ({
       footerLinks={footerLinks}
       phaseBanner={{
         phase: 'beta',
-        content: <span>Your opinions make things better. Share them in <a href="https://www.homeofficesurveys.homeoffice.gov.uk/s/D6XIF5/" target="_blank" rel="noopener noreferrer">this short survey</a>.</span>
+        feedbackUrl: '/feedback',
+        content: phaseBannerSurvey && <span>Your opinions make things better. Share them in <a href="https://www.homeofficesurveys.homeoffice.gov.uk/s/D6XIF5/" target="_blank" rel="noopener noreferrer">this short survey</a>.</span>
```
Comment (lennym) [inline ui/views/base.jsx:65]:
```suggestion
        content: phaseBannerSurvey
```

I'd just pass the content in as a prop from the implementation so it's less of a pain to modify in future.

I'd probably also rename the prop to `phaseBannerContent` while I'm there.

Comment (joefitter) [reply ui/views/base.jsx:65]: yeah, then it needs to be JSX and I can't set it in locals

Comment (joefitter) [reply ui/views/base.jsx:65]: without the markdown dance

Comment (lennym) [reply ui/views/base.jsx:65]: That sucks.

**Pull request merged**